audit/templates/audit-deployment.yaml

```diff
@@ -0,0 +1,138 @@
+####################################
+# Core Helm Template Configuration #
+####################################
+
+apiVersion: apps/v1
+kind: Deployment
+
+######################################
+# Basic Metadata for this Deployment #
+######################################
+
+metadata:
+  name: "{{ .Values.auditDeployment.meta.name }}"
```

Comment:
just like in accumulo, auditDeployment is redundant?

Reply:
agreed. next commit